multisig/tests/multisig_blackbox_test.rs

```diff
@@ -1050,3 +1154,1348 @@ fn multisig_non_board_member_sign_test() {
+        .returns(ReturnsResult)
+        .run();
+
+    assert_eq!(remaining_stake, BigUint::from(500u64));
+
+    state
+        .world
+        .tx()
+        .from(OWNER_ADDRESS)
+        .to(MULTISIG_ADDRESS)
+        .typed(multisig_proxy::MultisigProxy)
+        .withdraw_slashed_amount()
+        .run();
+
+    let remaining_slashed_amount = state
+        .world
+        .query()
+        .to(MULTISIG_ADDRESS)
+        .typed(multisig_proxy::MultisigProxy)
+        .slash_amount()
+        .returns(ReturnsResult)
+        .run();
+    assert_eq!(remaining_slashed_amount, BigUint::from(500u64));
+}
+
+#[test]
+fn test_withdraw_refund_fees_for_ethereum_success() {
+    let mut state = MultiTransferTestState::new();
+
+    let mut board: MultiValueEncoded<StaticApi, ManagedAddress<StaticApi>> =
+        MultiValueEncoded::new();
+    board.push(ManagedAddress::from(RELAYER1_ADDRESS.eval_to_array()));
+    board.push(ManagedAddress::from(RELAYER2_ADDRESS.eval_to_array()));
+    state
+        .world
+        .tx()
+        .from(OWNER_ADDRESS)
+        .typed(multisig_proxy::MultisigProxy)
+        .init(
+            MOCK_ESDT_SAFE_ADDRESS,
+            MULTI_TRANSFER_ADDRESS,
+            BRIDGE_PROXY_ADDRESS,
+            BRIDGED_TOKENS_WRAPPER_ADDRESS,
+            PRICE_AGGREGATOR_ADDRESS,
+            1_000u64,
+            500u64,
+            2usize,
+            board,
+        )
+        .code(MULTISIG_CODE_PATH)
+        .new_address(MULTISIG_ADDRESS)
+        .run();
+    state.mock_safe_deploy();
+
+    state
+        .world
+        .tx()
+        .from(OWNER_ADDRESS)
+        .to(MULTISIG_ADDRESS)
+        .typed(multisig_proxy::MultisigProxy)
+        .withdraw_refund_fees_for_ethereum(TokenIdentifier::from(WEGLD_TOKEN_ID))
+        .run();
+}
+
+#[test]
+fn test_withdraw_transaction_fees_success() {
+    let mut state = MultiTransferTestState::new();
+
+    let mut board: MultiValueEncoded<StaticApi, ManagedAddress<StaticApi>> =
+        MultiValueEncoded::new();
+    board.push(ManagedAddress::from(RELAYER1_ADDRESS.eval_to_array()));
+    board.push(ManagedAddress::from(RELAYER2_ADDRESS.eval_to_array()));
+    state
+        .world
+        .tx()
+        .from(OWNER_ADDRESS)
+        .typed(multisig_proxy::MultisigProxy)
+        .init(
+            MOCK_ESDT_SAFE_ADDRESS,
+            MULTI_TRANSFER_ADDRESS,
+            BRIDGE_PROXY_ADDRESS,
+            BRIDGED_TOKENS_WRAPPER_ADDRESS,
+            PRICE_AGGREGATOR_ADDRESS,
+            1_000u64,
+            500u64,
+            2usize,
+            board,
+        )
+        .code(MULTISIG_CODE_PATH)
+        .new_address(MULTISIG_ADDRESS)
+        .run();
+    state.mock_safe_deploy();
+
+    state
+        .world
+        .tx()
+        .from(OWNER_ADDRESS)
+        .to(MULTISIG_ADDRESS)
+        .typed(multisig_proxy::MultisigProxy)
+        .withdraw_transaction_fees(TokenIdentifier::from(WEGLD_TOKEN_ID))
```

Comment:
check balances

Reply:
same here. but great observation, i will check also on esdt_safe_blackbox_test.rs the balances. thanks

Reply:
then the checks for the balances also on esdt_safe_blackbox_test.rs is enough